multi-node.sh

```diff
@@ -46,7 +46,7 @@ runfabtests_script_builder()
     fi
     if [ ${PROVIDER} == "efa" ];then
         gid_c=$4
-        gid_s=$(cat /sys/class/infiniband/efa_0/ports/1/gids/0)
+        gid_s=$(/opt/amazon/efa/bin/fi_info -p efa | grep fabric: | head -n 1 | awk '{print $2}' | cut -d '-' -f 2)
```

Comment:
The sysfs approach tends to be more deterministic, which is why we avoided fi_info for this query. Using and rdma-core level query might be a bit better:

`ibv_devinfo -v | grep GID | awk '{print $3}'`

Reply:
Is it possible an instance have multiple rdma-core device and the first one is not EFA?

Reply:
I agree with you that ibv_devinfo is a better solution. 

I have update the PR